FATAL:   could not open image /bin/bash: image format not recognized
Traceback (most recent call last):
  File "/data/ndeo/bart/bart-hf-model/train-model-base.py", line 4, in <module>
    from transformers import AutoModelForSeq2SeqLM, Seq2SeqTrainingArguments, Seq2SeqTrainer, DefaultFlowCallback
  File "<frozen importlib._bootstrap>", line 1075, in _handle_fromlist
  File "/opt/conda/lib/python3.10/site-packages/transformers/utils/import_utils.py", line 1174, in __getattr__
    module = self._get_module(self._class_to_module[name])
  File "/opt/conda/lib/python3.10/site-packages/transformers/utils/import_utils.py", line 1184, in _get_module
    return importlib.import_module("." + module_name, self.__name__)
  File "/opt/conda/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/opt/conda/lib/python3.10/site-packages/transformers/training_args_seq2seq.py", line 21, in <module>
    from .training_args import TrainingArguments
  File "/opt/conda/lib/python3.10/site-packages/transformers/training_args.py", line 30, in <module>
    from .debug_utils import DebugOption
  File "/opt/conda/lib/python3.10/site-packages/transformers/debug_utils.py", line 21, in <module>
    import torch
  File "/opt/conda/lib/python3.10/site-packages/torch/__init__.py", line 1726, in <module>
    from torch import export as export
  File "/opt/conda/lib/python3.10/site-packages/torch/export/__init__.py", line 18, in <module>
    from torch.fx.passes.infra.pass_base import PassResult
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/passes/__init__.py", line 6, in <module>
    from . import reinplace
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/passes/reinplace.py", line 4, in <module>
    from torch._subclasses.fake_tensor import FakeTensorMode, FakeTensor
  File "/opt/conda/lib/python3.10/site-packages/torch/_subclasses/__init__.py", line 3, in <module>
    from torch._subclasses.fake_tensor import (
  File "/opt/conda/lib/python3.10/site-packages/torch/_subclasses/fake_tensor.py", line 35, in <module>
    from torch.utils._mode_utils import no_dispatch
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1002, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 945, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1439, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1411, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1577, in find_spec
  File "<frozen importlib._bootstrap_external>", line 161, in _path_isfile
  File "<frozen importlib._bootstrap_external>", line 153, in _path_is_mode_type
  File "<frozen importlib._bootstrap_external>", line 147, in _path_stat
KeyboardInterrupt
